packages/tabs/src/useTabs.ts

```diff
@@ -9,16 +9,46 @@ import { useState } from 'react';
 import { useSelection } from '@zendeskgarden/container-selection';
 import { generateId } from '@zendeskgarden/container-utilities';
 
-function requiredArguments(arg, argStr, methodName) {
+interface IUseTabsPropGetters {
+  getTabProps: <T>(options?: T & IGetTabPropsOptions) => T & IGetTabPropsOptions;
+  getTabPanelProps: <T>(options?: T & IGetTabPanelProps) => IGetTabPanelProps;
+  getTabListProps: <T>(options: T) => T & React.HTMLProps<any>;
+}
+
+interface IGetTabPropsOptions extends React.HTMLProps<any> {
+  index?: number;
+}
+
+interface IGetTabPanelProps extends React.HTMLProps<any> {
+  index?: number;
+  item?: any;
+}
+
+export interface IUseTabsReturnValue extends IUseTabsPropGetters {
+  selectedItem: string;
+  focusedItem: string;
+}
+
+export interface IUseTabsProps extends React.HTMLProps<any> {
```

Comment:
`useTabsProps` doesn't take any HTML props? What is this used for?

Reply:
Ah might of confused myself with the naming, but you're right. Will remove the extension to `React.HTMLProps`. Thanks!